packages/cli/src/commands/worker.ts

```diff
@@ -360,9 +355,42 @@ export class Worker extends BaseCommand {
 				},
 			);
 
-			server.listen(port, () => {
-				this.logger.info(`\nn8n worker health check via, port ${port}`);
-			});
+			let presetCredentialsLoaded = false;
+			const endpointPresetCredentials = config.getEnv('credentials.overwrite.endpoint');
+			if (endpointPresetCredentials !== '') {
```

Comment:
its basically a copy of code [here](https://github.com/n8n-io/n8n/blob/d6b2d8e800c2acaa05b724fdb39bcde0ab3fd61b/packages/cli/src/Server.ts#L1428) right? lgmt 👍

Reply:
yeah. I'd like to refactor worker to also use `AbstractServer`, but that's a much larger change that I'd like to avoid in this PR.